### PR TITLE
Reduce published package size by 164 KB by excluding tests

### PR DESCRIPTION
**Pull request by vladgolubev**
### Before:

```
140,837 bytes (283 KB on disk) for 39 items
```

<details> 
  <summary>Published package structure: </summary>

```
    .
    ├── CNAME
    ├── LICENSE
    ├── README.md
    ├── README.md.hb
    ├── _config.yml
    ├── index.js
    ├── middlewares.js
    ├── package.json
    ├── src
    │   ├── __tests__
    │   │   ├── isPromise.js
    │   │   └── middy.js
    │   ├── isPromise.js
    │   ├── middlewares
    │   │   ├── __tests__
    │   │   │   ├── cache.js
    │   │   │   ├── cors.js
    │   │   │   ├── doNotWaitForEmptyEventLoop.js
    │   │   │   ├── httpErrorHandler.js
    │   │   │   ├── index.js
    │   │   │   ├── jsonBodyParser.js
    │   │   │   ├── s3KeyNormalizer.js
    │   │   │   ├── urlencodeBodyParser.js
    │   │   │   ├── validator.js
    │   │   │   └── warmup.js
    │   │   ├── cache.js
    │   │   ├── cors.js
    │   │   ├── doNotWaitForEmptyEventLoop.js
    │   │   ├── httpErrorHandler.js
    │   │   ├── index.js
    │   │   ├── jsonBodyParser.js
    │   │   ├── s3KeyNormalizer.js
    │   │   ├── urlEncodeBodyParser.js
    │   │   ├── validator.js
    │   │   └── warmup.js
    │   └── middy.js
    └── typescript
        └── middy.d.ts

    5 directories, 33 files
```

</details>

### After: 

```
55,581 bytes (119 KB on disk) for 23 items
```

<details> 
  <summary>Published package structure: </summary>

```
.
├── LICENSE
├── README.md
├── README.md.hb
├── index.js
├── middlewares.js
├── package.json
├── src
│   ├── isPromise.js
│   ├── middlewares
│   │   ├── cache.js
│   │   ├── cors.js
│   │   ├── doNotWaitForEmptyEventLoop.js
│   │   ├── httpErrorHandler.js
│   │   ├── index.js
│   │   ├── jsonBodyParser.js
│   │   ├── s3KeyNormalizer.js
│   │   ├── urlEncodeBodyParser.js
│   │   ├── validator.js
│   │   └── warmup.js
│   └── middy.js
└── typescript
    └── middy.d.ts

3 directories, 19 files

```

</details>

Tiny improvement for https://github.com/middyjs/middy/issues/78

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
   "version": "0.7.0",
   "description": "The simple (but cool 😎) middleware engine for AWS lambda in Node.js",
   "main": "src/index.js",
+  "files": [
+    "src",
+    "!__tests__"
```
Comment (lmammino) [inline package.json]: `index.js` and `middlewares.js`?

Comment (vladgolubev) [reply package.json]: 😱 1 moment

File: package.json [hunk 2]
```diff
@@ -3,6 +3,12 @@
   "version": "0.7.0",
   "description": "The simple (but cool 😎) middleware engine for AWS lambda in Node.js",
   "main": "src/index.js",
+  "files": [
+    "src",
+    "index.js",
+    "middlewares.js",
+    "!__tests__"
```
Comment (lmammino) [inline package.json]: Also, we need `typescript` for people who want to use middy with typescript

**Comment by lmammino** [comment]
Let me version bump and get this merged 😉

**Comment by vladgolubev** [comment]
@lmammino BTW, shouldn't `"main": "src/index.js",` be `"main": "index.js",` ?

**Comment by lmammino** [comment]
@vladgolubev, good point, but that' will trigger reading an extra file by default.

The only reason why it's there is to support:

```javascript
const middy = require('middy') // or
const middy = require('middy/index') // as similar to
const middlewares = require('middy/middlewares') // ...
``` 

To be fair it is not very useful and adds unneeded boilerplate to support a rather unnecessary use case... so we should probably remove it, but let's get this merged for now.

**Pull request merged**